File: acmp.ru/[0801] - [0900]/[0854] Кондиционер.cpp
#include <iostream>
#include <cstring>
using namespace std;

int main() {
    int t1, t2;
    string mode;

    cin >> t1 >> t2 >> mode;

    if (mode == "freeze")
        cout << min(t1, t2) << endl;
    else if(mode == "heat")
        cout << max(t1, t2) << endl;
    else if(mode == "auto")
        cout << t2 << endl;
    else
        cout << t1 << endl;

    return 0;
}

File: acmp.ru/[0801] - [0900]/[0854] Кондиционер.py
t1, t2 = [int(i) for i in input().split()]
mode = input()

if mode == 'freeze':
    print(min(t1, t2))
elif mode == 'heat':
    print(max(t1, t2))
elif mode == 'auto':
    print(t2)
else:
    print(t1)
